hawkular-inventory-rest-api/pom.xml

```diff
@@ -59,6 +59,18 @@
     </dependency>
 
     <dependency>
+      <groupId>org.hawkular.inventory</groupId>
+      <artifactId>hawkular-inventory-security-spi</artifactId>
+      <version>${project.version}</version>
+    </dependency>
+
+    <dependency>
```

Comment:
No, rest API has a real compile time dependency on `hawkular-inventory-security-spi` and therefore the dependency on `hawkular-inventory-security-spi` cannot be moved to `hawkular-inventory-dist`